dashboard/app/views/layouts/_thank_donors_interstitial.html.haml

```diff
@@ -0,0 +1,24 @@
+
+#thank-donors-modal.modal{style: 'display: none;'}
+  %button#close-thank-donors-dialog.close{'data-dismiss' => 'modal', 'aria-label' => 'Close'}
+    %span{'aria-hidden' => 'true'}= '&times;'.html_safe
+  .modal-dialog
+    .modal-content.no-modal-icon
+      %h2.donor-header= t('donor_interstitial.title') + '...'
+      %p= t('donor_interstitial.subtitle', donors_url: CDO.code_org_url('/about/donors'), markdown: :inline).html_safe
+      %h2.donor-subheader= t('donor_interstitial.global_supporters')
+      .clear
+      .scroll-box
+        .donor-container
+          - Donor.where(level: %w(platinum diamond)).order(:weight).pluck(:name).each do |donor|
+            .donor= donor
+      - if current_user.student?
```

Comment:
I might be overly paranoid here, but I'd add a comment at the top of this partial mentioning that it assumes the presence of a signed-in user.